src/components/InventoryManager/ListingSteps/UnitsStep.js

```diff
@@ -0,0 +1,396 @@
+                {listingRecord['individual produce unit'].toUpperCase() || 'INDIVIDUAL PRODUCE UNIT'}
+              </Typography>
+              &nbsp;
+              <Typography display="inline" className={classes.text}>PER</Typography>
+              &nbsp;
+              <Typography display="inline" className={classes.labels}>
+                {listingRecord['grouped produce type'].toUpperCase() || 'GROUPED PRODUCE TYPE'}
+              </Typography>
+            </Grid>
+          </Grid>
+        </Grid>
+        <Grid container item xs={12}>
+          <Grid container spacing={1} item xs={12}>
+            <Grid item xs={12}>
+              <Typography className={classes.subheading}>
+                Pounds per Grouped Produce Type
+              </Typography>
+            </Grid>
+            <Grid container item xs={12} alignItems="center">
+              <ListingInputField
+                id="standard-number"
+                name="lbs per grouped produce type"
+                type="number"
+                onChange={onChangeField}
+                val={listingRecord['lbs per grouped produce type']}
+                onButtonClick={onButtonClick}
+              />
+              <Typography display="inline" className={classes.text}>
+                LBS PER
+              </Typography>
+              &nbsp;
+              <Typography display="inline" className={classes.labels}>
+                {listingRecord['grouped produce type'].toUpperCase() || 'GROUPED PRODUCE TYPE'}
+              </Typography>
+            </Grid>
+          </Grid>
+        </Grid>
+        <Grid item container xs={12}>
+          <Typography className={classes.heading}>{questions[1]}</Typography>
+        </Grid>
+        <Grid item container xs={12}>
+          {/* TODO : reset master unit count when checbox unchecked */}
+          <Typography className={classes.text}>
+            <GreenCheckbox
+              checked={listingRecord['has master units']}
+              onChange={() => onSelect('has master units', !listingRecord['has master units'])}
+            />
+            Yes, I do have master units.
+          </Typography>
+        </Grid>
+        {(listingRecord['has master units'])
+        && (
+        <Grid container item xs={12}>
+          <Grid container spacing={1} item xs={12}>
+            <Grid item xs={12}>
+              <Typography className={classes.subheading}>
+                Quantity
+              </Typography>
+            </Grid>
+            <Grid container item xs={12} alignItems="center">
+              <ListingInputField
+                id="standard-number"
+                name="grouped produce type per master unit"
+                type="number"
+                onChange={onChangeField}
+                val={listingRecord['grouped produce type per master unit']}
+                onButtonClick={onButtonClick}
+              />
+              <Typography display="inline" className={classes.labels}>
```

Comment:
nitpick, but can you have an 's' after the grouped produce type? So like sacks per master crate instead of sack per master crate